pom.xml

```
@@ -270,6 +270,7 @@
     <formatter.config>eclipse-formatter-config-2space.xml</formatter.config>
     <gcu.product>${project.name}</gcu.product>
     <maven.min-version>3.2.5</maven.min-version>
+    <module.name>org.mybatis</module.name>
```

There cannot be multiple modules with the same name, so we must declare unique name in all projects.

```suggestion
    <module.name>org.mybatis.parent</module.name>
```

---

I figured that would be a good default but if any are missed it would be problematic.  I'll make the switch.